packages/Panel/src/Panel.js

```diff
@@ -192,52 +209,60 @@ const propTypes = {
   /** Y offset that is passed down from <Panel.Group> */
   groupOffsetY: PropTypes.number,
 
-  /** Control the compactness of the side panel */
+  /** The height of the open Panel (when slide in from bottom) */
+  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+  /** Control the compactness of the Panel */
   isCompact: PropTypes.bool,
 
   /** Render the panel inline */
   isInline: PropTypes.bool,
 
-  /** Control the visibility of the side panel. This prop makes the side panel appear */
+  /** Control the visibility of the Panel. This prop makes the Panel appear. */
   isOpen: PropTypes.bool.isRequired,
 
-  /** Control if the side panel slides from the left */
-  isSlideFromLeft: PropTypes.bool,
-
   /** Modify the look of the Panel */
   kind: PropTypes.oneOf([Panel.types.kind.DEFAULT, Panel.types.kind.CHILD, Panel.types.kind.PRIMARY]),
 
-  /** Control y offset of the panel */
-  offsetY: PropTypes.number,
+  /** Control offset of the Panel. Only use 'top' when sliding in from the left or right. Only use 'left' or 'right' when sliding in from the bottom. */
+  offset: PropTypes.shape({ top: PropTypes.number, left: PropTypes.number, right: PropTypes.number }),
 
-  /** Callback once the panel has been closed event */
+  /** Callback once the Panel has been closed event */
   onAfterClose: PropTypes.func,
 
-  /** Callback once the panel has been opened event */
+  /** Callback once the Panel has been opened event */
   onAfterOpen: PropTypes.func,
 
-  /** Callback triggered when the side panel needs to be close */
+  /** Callback triggered when the Panel needs to be close */
   onClose: PropTypes.func,
 
-  /** The width of the open panel. */
+  /** Control where the Panel slides in from */
+  slideFrom: PropTypes.oneOf([
+    Panel.slideFromDirections.RIGHT,
```

Comment:
This should follow the convention of `Panel.types.slide.RIGHT`

just add a declaration inside of Panel.js and do

```js	
Panel.types = {
  slide: {
    RIGHT: "right",
    LEFT: "left",
    etc
  }
}
```